README.md

```diff
@@ -12,6 +12,11 @@ resource "aws_route53_zone" "default" {
 module "cert" {
   source = "github.com/azavea/terraform-aws-acm-certificate?ref=0.1.0"
 
+  providers = {
```

Comment:
Leaving this comment here because there didn't seem to be a great place for it:

I'd be OK with breaking backwards compatability provided we added something like this to alert people if they tried to use the module with < 0.11:

```hcl
terraform {
  required_version = ">= 0.11.0"
}
```